### DISCUSSION TITLE
Using maplibre-gl-js on commercial site

### DISCUSSION THREAD

**Discussion by ticketationguy** [maplibre/maplibre-gl-js#2737] [Q&A]
Hi,
We are willing to move to maplibre-gl-js and wondered if we can just use it? Or should we add some credits/notices somewhere?
Thanks

**Comment by HarelM** [comment]
See
https://github.com/maplibre/maplibre-gl-js/blob/main/LICENSE.txt
You should add credit in general.
This is usually done by using a tool like webpack etc that creates a 3rdpartylicenses.txt file.

Also giving credit in a more visible way is always welcome.

**Reply by ticketationguy** [reply]
@HarelM Any example for such a site? Thanks!

**Reply by HarelM** [reply]
https://Israelhiking.osm.org.il/

**Reply by ticketationguy** [reply]
Thanks @HarelM byt I cant find that file there. Can you guide me where can I see it? thanks

**Reply by HarelM** [reply]
https://israelhiking.osm.org.il/3rdpartylicenses.txt

**Reply by HarelM** [reply]
Seems like the maplibre-gl npm package has a missing license file... :-(

**Reply by HarelM** [reply]
Relevant issue can be found here: #2865. It shouldn't be hard to solve this though if anyone is interested.

**Reply by ticketationguy** [reply]
@HarelM so it means we cant give maplibre credit right now?

**Reply by HarelM** [reply]
You can always give credit, but currently there's a tooling bug that is described in the issue above.

**Reply by ticketationguy** [reply]
@HarelM can we start using with no credit? 
I am just checking in order to be on the safe side legal wise.

**Reply by HarelM** [reply]
You need to give credit, either via automatic tools or manually.

**Reply by ticketationguy** [reply]
@HarelM how do we do it manually? 
just to mention it in 3rdpartylicenses.txt file? do you have any example for it? Thanks

**Reply by HarelM** [reply]
Yes, copy the content of the license file from this repo into your 3rd...txt file.

**Comment by wipfli** [comment]
If you depend on MapLibre and want to make sure that it continues to exist in the future, consider becoming a sponsor at https://opencollective.com/maplibre
